src/Tasks.UnitTests/CSharpParserUtilitites_Tests.cs

```diff
@@ -333,18 +390,34 @@ namespace i
 
 
 
+        [Theory]
+        [InlineData(@"
+namespace n2
+// namespace n1
+{ class c {} }")]
+        [InlineData(@"
+namespace n2;
+// namespace n1
+class c {}")]
+        public void Regress_Mutation_SingleLineCommentsShouldBeIgnored(string fileContents)
+        {
+            AssertParse(fileContents, "n2.c");
+        }
+
         [Fact]
-        public void Regress_Mutation_SingleLineCommentsShouldBeIgnored()
+        public void FileScoped_Namespace()
```

Comment:
Is there a point to this standalone test now? isn't it covered by one of the ones you changed above?

Reply:
After the recent edits adding comments for every instance of a filescoped namespace, we can get rid of this one.